pkg/queue/bounded_queue.go

```diff
@@ -165,10 +170,13 @@ func (q *BoundedQueue) Resize(capacity int) bool {
 	swapped := atomic.CompareAndSwapPointer((*unsafe.Pointer)(unsafe.Pointer(&q.items)), unsafe.Pointer(q.items), unsafe.Pointer(&queue))
 	if swapped {
 		// start a new set of consumers, based on the information given previously
-		q.StartConsumers(q.workers, q.consumer)
+		q.StartConsumers(int(q.workers), q.consumer)
```

Comment:
q.workers is already `int`

Reply:
Did you spot it using a different linter?

Reply:
no, just paying attention

Reply:
We should plug your brain in the CI, as our linters didn't catch it.